Client.xcodeproj/project.pbxproj

```diff
@@ -14281,7 +14273,6 @@
 		};
 		43EFA63D2786562600400DF0 /* Places */ = {
 			isa = XCSwiftPackageProductDependency;
-			package = 1BE9ECAA271765C800D729BB /* XCRemoteSwiftPackageReference "rust-components-swift" */;
```

Comment:
This got deleted when the above extra reference to rust-components-swift was removed... I don't see a `Places` target in Xcode so I'm not sure what this is doing here 🙃